src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+
+import {
+  Pagination as CorePagination,
+  PaginationProps as CorePaginationProps,
+} from 'wix-ui-core/pagination';
+import { ChevronLeftLarge, ChevronRightLarge } from 'wix-ui-icons-common';
+
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+
+import { DoubleChevronLeftLarge, DoubleChevronRightLarge } from './icons';
+import styles from './Pagination.st.css';
+
+export interface PaginationProps extends CorePaginationProps {
+  /** Justify content */
+  alignment?: 'left' | 'center' | 'right';
```

Comment:
no need to redefine all properties from `CorePaginationProps` just extend core props with `alignment` and `mobileView`

Reply:
`alignment` not optional anymore, at least as long as you are defining it in `defaultProps`